presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestJdbcResultSet.java

```diff
@@ -97,10 +103,11 @@ public void testDuplicateColumnLabels()
     public void testObjectTypes()
             throws Exception
     {
-        String sql = "SELECT 123, 12300000000, REAL '123.45', 0.1, true, 'hello', 1.0E0 / 0.0E0, 0.0E0 / 0.0E0, ARRAY[1, 2], cast('foo' as char(5)), DECIMAL '0.1'";
+        String sql = "SELECT 123, 12300000000, REAL '123.45', 0.1, true, 'hello', 1.0E0 / 0.0E0, 0.0E0 / 0.0E0, ARRAY[1, 2], cast('foo' as char(5)), DECIMAL '0.1', " +
+                "DATE '2018-02-13', TIME '09:39:05', TIME '09:39:07 +01:00', TIMESTAMP '2018-02-13 13:14:15.123', TIMESTAMP '2018-02-13 13:14:15.227 Europe/Warsaw'";
```

Comment:
I realize the others are squished together, but maybe put each one on a separate line for readability.

Reply:
i didn't resist the temptation and refactored this test method so that individual cases are separate